processed 17 tasks

task 1 'run'. lines 20-29:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x2::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 2 'run'. lines 30-39:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x3::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 4 'run'. lines 60-69:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x5::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 5 'run'. lines 70-79:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x42::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 7 'run'. lines 98-107:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x7::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 8 'run'. lines 108-117:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x8::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 10 'run'. lines 138-147:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x10::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 11 'run'. lines 148-157:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x11::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 12 'run'. lines 158-167:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x12::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 14 'run'. lines 188-197:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x14::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 15 'run'. lines 198-207:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x15::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}

task 16 'run'. lines 208-217:
Error: Function execution failed with VMError: {
    major_status: ARITHMETIC_ERROR,
    sub_status: None,
    location: 0x16::m,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 2)],
}
